x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/defense_evasion_adding_the_hidden_file_attribute_with_via_attribexe.json

```diff
@@ -61,5 +61,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 11
+  "version": 13
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950